packages/layout/src/Border2.css

```diff
@@ -6,22 +10,17 @@
 }
 
 .layout_Border2 > .body > .center {
-    flex: 3 1 60%;
+    flex: 1;
     -ms-flex: 3 1 auto;
```

Comment:
should the ms specific be changed?

Reply:
If IE11 is our target IE browser then it should actually be removed.

Reply:
Yup - I do know that the defaults in IE are diff to chrome:
IE:  "0 0 auto" 
Chrome:  "0 1 auto"

So you may need something?